dp/generate_climos.py

```diff
@@ -484,7 +498,8 @@ def split_on_variables(climo_file, var_names):
         return [climo_file]
 
 
-def update_metadata_and_time_var(input_file, t_start, t_end, operation, climo_filepath):
+def update_metadata_and_time_var(input_file, outdir, filepath, t_start, t_end, operation, climo_filepath, t_start_generate_climos, t_end_generate_climos, 
```

Comment:
I'm wondering if the approach here could be simplified: What if `generate_climos` just inserted something into the history before kicking off all the `cdo` operations, and finished up by inserting something afterwards? We would need to find a file in which to add the "start" history item; do you think that the initial selection of the temporal subset would be a simple place to do this?